server/src/controllers/ChapterUser/resolver.ts

```diff
@@ -83,7 +83,41 @@ export class ChapterUserResolver {
       where: {
         user_id_chapter_id: { chapter_id: chapterId, user_id: ctx.user.id },
       },
-      include: includes,
+      include: chapterIncludes,
+    });
+  }
+
+  @Mutation(() => ChapterUser)
+  async leaveChapter(
+    @Arg('chapterId', () => Int) chapterId: number,
+    @Ctx() ctx: Required<ResolverCtx>,
+  ): Promise<ChapterUser | null> {
+    await prisma.event_users.deleteMany({
```

Comment:
As mentioned above, this resolver will need to cancel the Google Calendar invites for these events.